packages/unenv-preset/build.config.ts

```diff
@@ -0,0 +1,19 @@
+import { defineBuildConfig } from "unbuild";
+
+export default defineBuildConfig({
+	declaration: true,
+	rollup: {
+		emitCJS: true,
+	},
+	entries: [
+		"src/index",
+		{ input: "src/runtime/", outDir: "runtime", format: "esm" },
```

Comment:
Can we output to `dist/runtime` and then fix up the package.json exports, which would also allow us to remove the `.gitignore` change.

Reply:
Yes, I can change to output to 2 different dist subfolders